### PR TITLE
Add airbnb-base to eslint config (closes #6986)

### LINKED ISSUE #6986: Use eslint-config-airbnb-base for project initialization
When running `eslint --init` one of the options is to use airbnb style as a base. If chosen, we install [eslint-config-airbnb](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) which requires `eslint-plugin-react` among others.

I think we should install [eslint-config-airbnb-base](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/package.json) instead.

### PR DESCRIPTION
**Pull request by jacobp100**
**What issue does this pull request address?**
Adds airbnb base config for projects that use their style guide, but aren't using React

**What changes did you make? (Give an overview)**
Added airbnb-base to config

**Is there anything you'd like reviewers to focus on?**
Nope!

### CODE REVIEW

**Comment by platinumazure** [comment]
We have a discussion in another issue about whether we should replace airbnb with airbnb-base, or adding to it.

@ljharb Any recommendations? Is there a strong desire to be able to initialize both configurations in ESLint, as far as you can tell?

**Comment by ljharb** [comment]
Both seems better to me, but if we only want one, it'd be the base config.

File: lib/config/config-initializer.js [hunk 1]
```diff
@@ -305,7 +306,7 @@ function promptUser(callback) {
             type: "list",
             name: "styleguide",
             message: "Which style guide do you want to follow?",
-            choices: [{name: "Google", value: "google"}, {name: "Airbnb", value: "airbnb"}, {name: "Standard", value: "standard"}],
+            choices: [{name: "Google", value: "google"}, {name: "Airbnb", value: "airbnb"}, {name: "Airbnb Base (no react)", value: "airbnb"}, {name: "Standard", value: "standard"}],
```
Comment (vitorbal) [inline lib/config/config-initializer.js]: I think you need to change the value of "AirbnbBase (no react)" to `airbnb-base`, otherwise this won't work.

Comment (kaicataldo) [inline lib/config/config-initializer.js]: I also believe "React" should be capitalized

**Comment by vitorbal** [comment]
LGTM

**Comment by nzakas** [comment]
I don't think this is the correct way to do this. New users will not understand the difference between "Airbnb" and "Airbnb (no React)". We should let them select Airbnb and then later, depending if they say they are using React, install the correct config.

**Comment by kaicataldo** [comment]
Hey @jacobp100, friendly ping to see if you needed any clarification on the comment above and if you're still able to work on this. Thanks!

**Comment by kaicataldo** [comment]
@jacobp100 One more friendly ping on this

**Comment by jacobp100** [comment]
Hey, sorry. I'll look at it when I get time, and will submit a new pr.

**Comment by kaicataldo** [comment]
No worries! Totally understand and appreciate the contribution regardless.

**Pull request closed without merging**